### ISSUE TITLE
Is it possible to modify tesstrain.sh/tesstrain_util.sh for using images I owned(I made box/tif manually)

### ISSUE THREAD

**Issue by godofcheerup** [tesseract-ocr/tesseract#1445]
------------------------

### Environment

* **Tesseract Version**:  tesseract 4.00 Beta.1
* **Commit Number**:  
* **Platform**:  ubuntu 16.04.03 LTS

### Current Behavior: 

I want to train multiple images (which I owned) by using tesseract 4.00 Beta.1 

I succeed in training one image (by not using tesstrain.sh) I made box/tif , lstmf , unicharset(extractor_unicharset) , But, one tif(one image + image - in parallel ) I made box/tif, lstmf ,

But It give me an error LSTM: Training - Compute CTC targets failed! #591

In conclusion , I want to use tesstrain.sh, for my training, Could you tell me more details , to modify tesstrain.sh?

**Comment by Shreeshrii** [comment]
Training from images not supported for tesseract 4.0.0-beta.1.

@zdenop Please close.

**Comment by wrznr** [comment]
You might want to have a look at https://github.com/OCR-D/ocrd-train
